static/elements/form-field-specs.js

```diff
@@ -357,20 +385,18 @@ export const ALL_FIELDS = {
       designs? Double check that your experiment makes sense given that 
       a large developer (e.g. a Google product or Facebook) likely 
       can't use it in production due to the limits enforced by origin 
-      trials.\n\nIf Intent to Extend Origin Trial, highlight new/different 
-      areas for experimentation. Should not be an exact copy of goals 
+      trials.\n\nIf you send an Intent to Extend Origin Trial, highlight 
+      areas for experimentation. They should not be an exact copy of the goals 
```

Comment:
Why take out "new/deferent"?  I assume those words were used oringally because extending an experiment should include be something that is different from the original experiment.

Reply:
Joe's change removed the "new/different" text, probably because it is moderately unessential.  But perhaps there is nothing new or different, but they just need another round of experimentation, so maybe it should say "highlight the reason for the extension".